classes/oc/captcha.php

```diff
@@ -84,7 +84,7 @@ public static function check($name = '', $ajax = FALSE)
         //d(Session::instance()->get('captcha_'.$name));
         //d(Session::instance()->get('captcha_'.$name) == strtolower(core::post('captcha')));
 
-        if (core::config('advertisement.captcha') == FALSE) // Captchas are disabled
+        if (core::config('advertisement.captcha') == FALSE AND core::config('general.captcha') == FALSE) // Captchas are disabled
```

Comment:
But `classes/oc/captcha.php` it's on common.

I think `if (core::config('advertisement.captcha') == FALSE AND core::config('general.captcha') == FALSE)` is the right conditional for both.

**OC**

core::config('general.captcha’) is always NULL.

*Case 1*

advertisement.captcha = 1 then is false that (1 == FALSE and NULL == FALSE).

*Case 2*

advertisement.captcha = 0 then is true that (0 == FALSE and NULL == FALSE).

**OE**

core::config('advertisement.captcha’) is always NULL.

*Case 1*

general.captcha = 1 then is false that (NULL == FALSE and 1 == FALSE).

*Case 2*

general.captcha = 0 then is true that (NULL == FALSE and 0 == FALSE).